java/src/jmri/jmrix/roco/z21/Z21LnStreamPortController.java

```diff
@@ -34,7 +34,7 @@ public void configure() {
         this.getSystemConnectionMemo().setLnTrafficController(packets);
         // do the common manager config
         this.getSystemConnectionMemo().configureCommandStation(commandStationType,
-                mTurnoutNoRetry, mTurnoutExtraSpace, false); // never transponding
+                mTurnoutNoRetry, mTurnoutExtraSpace, false, false); // never transponding
```

Comment:
This should be configurable as well.

The Roco Z21 has a LocoNet connection and JMRI should be able to send interrogate request to that connection as well.

Reply:
Currently the roco doesnt send the queries, which may be due to a programming error, but no ones complaining.

Reply:
> Currently the roco doesnt send the queries, which may be due to a programming error, but no ones complaining.

@pabender 
Do you know why this controller doesn't send the queries?

Reply:
The errors were 
The local variable rcvThread is hiding a field from type LnPacketizer	LnStreamPortPacketizer.java	/jmri/java/src/jmri/jmrix/loconet/streamport	line 154	Java Problem
The local variable xmtThread is hiding a field from type LnPacketizer	LnStreamPortPacketizer.java	/jmri/java/src/jmri/jmrix/loconet/streamport	line 144	Java Problem
(there are another bizzillion of these !)
Fixed that, and forced the interrogateAtStart to "No", using the same reasoning as the turnout handling.
There is no configuration for the loconet (which is only a Throttle Bus), which is really just an add on, as is the XPressnet to the Roco.

Reply:
> There is no configuration for the loconet (which is only a Throttle Bus), which is really just an add on, as is the XPressnet to the Roco.

This is actually a misinterpretation of the code.

The Z21 tunnels both XpressNet and LocoNet through its UDP based network protocol.  The tunneling was done to prevent a complete re-write of the respective protocols.

The two Stream port controllers used with The Z21 handle processing the Z21 messages for their respective buses after the Z21 headers are removed.

Both the XPressNet and Loconet connections translate to real interface buses on the Z21.  I.e. the Z21 has a fully functional LocoNet and XPressNet connection.